wgsl/index.bs

```diff
@@ -535,6 +535,20 @@ literal_or_ident
     The third parameter, if provided, specifies the z dimension, otherwise is assumed to be 1.
     Each dimension must be at least 1 and at most an upper bound specified by the WebGPU API.
 
+  <tr><td><dfn noexport dfn-for="attribute">`precise_math`</dfn>
+    <td>*None*
+
+    Indicates that the arithmetic computations in the function need to be performed with
```

Comment:
I have trouble with the word "precision" here, because that means "with more bits represented". 
(Never mind the "precise" part of the attribute name, inherited from GLSL.  It's good to reuse the GLSL word.)

Also, this should be constrained to floating point, I think.


How about:

> Indicates that the floating point arithmetic computations in the function should be performed
>  * without [=reassociation/reassociating=] subexpressions
> * while preserving infinities, NaNs, and signed zeroes

> Apply this attribute when the correctness of the function is numerically sensitive, and it is acceptable to incur potential performance loss when forbidding such optimizations.

blah blah blah?

Reply:
Thank you! 
I took the liberty of modifying this a bit more. Let me know if it needs more fixing!